infrastructure/testharness.js/single-page-test-silent-before-results.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE HTML>
+<html>
+<head>
+<title>Harness does not modify DOM prior to completion</title>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+</head>
+<body><script>
+'use strict';
+/**
+ * This property is useful for tests which concern events related to the
+ * initial rendering of the document, e.g. paint-timing.
+ */
+setup({single_test: true});
+addEventListener('load', function() {
+  step_timeout(function() {
+    var childNodes = Array.prototype.slice.call(document.body.childNodes);
```

Comment:
`Array.from` would be OK I think, as long as it’s supported in stable versions of all the browsers we currently run.

Reply:
Or actually... just don’t turn it into an array and use it as a NodeList?

Reply:
Definitely